e2etest/zt_newe2e_remove_test.go

```diff
@@ -290,3 +290,41 @@ func (s *RemoveSuite) Scenario_RemoveBlobsWithExcludePath(svm *ScenarioVariation
 	ValidateResource[ContainerResourceManager](svm, src, ResourceDefinitionContainer{}, true)
 	ValidateDoesNotContainError(svm, stdOut, []string{"unknown flag: --exclude-path"})
 }
+
+// Scenario_RemoveFilesWithSpecialChars validates that we correctly remove special chars and do not decode them prematurely
+func (s *RemoveSuite) Scenario_RemoveFilesWithSpecialChars(svm *ScenarioVariationManager) {
+	if !svm.Dryrun() {
+		src := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.File()),
+			ResourceDefinitionContainer{})
+
+		fullList := []string{
```

Comment:
CreateResource with a FlatResourceMapping might make sense with this; it's not effectually different though.